Header mismatch when running 'Simple Audio Recognition'

tensorflow: 1.4, ubuntu 17.10, python3.6 anaconda, cuda 8.0, cudnn 6.0
Going with Simple Audio Recognition, I met an error when running
python tensorflow/examples/speech_commands/train.py --data_dir=tensorflow/examples/speech_commands/train/audio
error:
Traceback (most recent call last):                                                                                                                            
  File "/home/jihao/.conda/envs/tf_cpu/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1323, in _do_call                               
    return fn(*args)                                                                                                                                          
  File "/home/jihao/.conda/envs/tf_cpu/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1302, in _run_fn                                
    status, run_metadata)                                                                                                                                     
  File "/home/jihao/.conda/envs/tf_cpu/lib/python3.6/site-packages/tensorflow/python/framework/errors_impl.py", line 473, in __exit__                         
    c_api.TF_GetCode(self.status.status))                                                                                                                     
tensorflow.python.framework.errors_impl.InvalidArgumentError: Header mismatch: Expected RIFF but found \udc87\udc9b\udc8f\udc8c                               
         [[Node: DecodeWav = DecodeWav[desired_channels=1, desired_samples=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](ReadFile)]]